apps/signer/gst-plugin/gstsigning.c

```diff
@@ -72,6 +77,45 @@ setup_signing(GstSigning *signing, GstCaps *caps);
 static gboolean
 terminate_signing(GstSigning *signing);
 
+static void
+gst_signed_video_get_property(GObject *object, guint prop_id, GValue *value, GParamSpec *pspec)
```

Comment:
Change name to `gst_signing_get_property`. The element name is `signing` and that should be reflected in the function names. This comment holds for other places as well.